docs/development/lint.zh.md

```diff
@@ -1,3 +1,15 @@
-# Lint
+# 代码检查
 
-todo
+
+项目使用`golangci-lint` ([official website](https://golangci-lint.run/), [GitHub](https://github.com/golangci/golangci-lint)) 做代码检查，也是Go的代码检查工具集。
+
+也支持与 Github Actions 工作流集成。
+
+- 记录默认启用的检查项的 [文档](https://golangci-lint.run/usage/linters/) 。
+- 也可以和 [IDE](https://golangci-lint.run/usage/integrations/) 集成。
+- 也可以在 [本地运行](https://golangci-lint.run/usage/quick-start/) 。
+
+除此之外，我们使用 [Go Report Card](https://goreportcard.com/report/github.com/devstream-io/devstream) ，为你的开源 Go 代码生成质量报告。
+
+在项目的README.md中，有代码质量标志 [![Go Report Card](https://goreportcard.com/badge/github.com/devstream-io/devstream)](https://goreportcard.com/report/github.com/devstream-io/devstream) 。
```

Comment:
Good!

Reply:
> I will submit as soon as possible after the modification is complete, should I submit a new pr

No, you  needn't. Just git push more commits and the new commits will be added to this pr.